ecc/goldilocks/constants.go

```diff
@@ -70,4 +76,5 @@ var (
 		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
 		0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
 	}
+	errInvalidDecoding = errors.New("invalid decoding")
```

Comment:
Probably, this should be 'invalid decaf decoding', as there can be other decodings..